pkg/operator/ceph/cr_manager.go

```diff
@@ -156,11 +158,22 @@ func (o *Operator) startCRDManager(context context.Context, mgrErrorCh chan erro
 
 	// Set up a manager
 	mgrOpts := manager.Options{
-		LeaderElection:     false,
-		Namespace:          o.config.NamespaceToWatch,
-		MetricsBindAddress: "0",
-		Scheme:             scheme,
-		CertDir:            certDir,
+		LeaderElection: false,
+		Metrics: metricsserver.Options{
+			// BindAddress is the bind address for controller runtime metrics server default is 8080. Since we don't use the
+			// controller runtime metrics server, we need to set the bind address 0 so that port 8080 is available.
+			BindAddress: "0",
+		},
+		Scheme: scheme,
+		WebhookServer: webhook.NewServer(webhook.Options{
+			CertDir: certDir,
+		}),
+	}
+
+	if o.config.NamespaceToWatch != "" {
+		mgrOpts.Cache = cache.Options{
+			DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: {}},
+		}
```

Comment:
```suggestion
	if o.config.NamespaceToWatch != "" {
		mgrOpts.Cache = cache.Options{
			DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: {}},
		}
```

It is recommended to use the controller namespace only when watching of one name space, watching all it's recommended to not use the controller namespace. 
```
So, if you would like to watch all you should not use this one and see that it is the default behaviour of the manager.

```
from https://github.com/kubernetes-sigs/controller-runtime/issues/1006#issuecomment-713695905